ddl/column.go

```diff
@@ -1490,81 +1594,24 @@ func (w *worker) doModifyColumn(
 }
 
 func adjustColumnInfoInModifyColumn(
```

Comment:
Do we need to unify this function name and [adjustTableInfoAfterModifyColumn](https://github.com/pingcap/tidb/pull/33631/files#diff-42e0180cdc8c14def674a9e45d908d6700f6a8ccba0f9ff4b14b27c0daa49859R1083)? Maybe this function named like `adjustTableInfoAfterModifyColumnWithoutData`, another named `adjustTableInfoAfterModifyColumnWithData`.